modules/lambda-kms/main.tf

```diff
@@ -0,0 +1,18 @@
+data "template_file" "kms_policy" {
```

Comment:
The `template_file` data resource for IAM policies is an older way of doing this. Can you switch this to use the newer [`iam_policy_document`](https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html) data resource? There are a few examples in this repo of how to use it such as [this one](https://github.com/rackerlabs/xplat-terraform-modules/blob/master/modules/lambda/main.tf#L3).